deployment/clusterCmd.py

```diff
@@ -38,6 +38,14 @@
 
 class ClusterCmd():
     def register(self, parser):
+
+        logger.warning("Kubernetes deployment in paictl maybe be deprecated in the future!")
+        logger.warning("We highly recommend deploying kubernetes with kubespray.")
+        logger.warning("If you wanna deploy k8s with kubespray, please refer to https://github.com/microsoft/pai/tree/master/contrib/kubespray")
+        logger.warning("paictl cluster command will sleep here for 10 seconds.")
+        logger.warning("If you wanna quit this process, please type ctrl + c")
+        time.sleep(10)
```

Comment:
Instead of sleep 10 second, I suggest to explicitly ask user to input "YES" to proceed to the next step. Default to "NO".

Reply:
This will have an influne on Jenkins. If we add additional input, we will have to change jenkins script.

Reply:
could you change the Jenkins accordingly?

Reply:
@ydye you could change lines [here](https://github.com/microsoft/pai/blob/master/tests/jenkins/stage_deploy.sh#L45-L52) or [there](https://github.com/microsoft/pai/blob/master/tests/jenkins/stage_clean.sh#L30-L34), just add `echo "Y" | python ...`

Reply:
Done